Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1024,6 +1040,21 @@
   - MedicalDoctorPDA
   - SeniorPhysicianPDA
 
+- type: loadoutGroup
+  id: MedicalDoctorGloves
+  name: loadout-group-medical-doctor-gloves
+  minLimit: 0
+  loadouts:
+  - GlovesLatex
+  - GlovesNitrile
```

Comment:
why arent these just one `loadoutGroup` called like `DoctorGloves` that all of the jobs have instead of it being duplicated 4 times